vite.config.ts

```diff
@@ -1,63 +1,51 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-/// <reference types="vitest" />
-
 import path from "path";
 
-import tailwindcss from "@tailwindcss/vite";
-import react from "@vitejs/plugin-react";
 import { visualizer } from "rollup-plugin-visualizer";
-import { defineConfig, type PluginOption } from "vite";
+import { type PluginOption } from "vite";
 import dts from "vite-plugin-dts";
 
-export default defineConfig({
-  plugins: [
-    react(),
-    tailwindcss(),
-    visualizer({
-      open: true,
-      gzipSize: true,
-      brotliSize: true,
-      filename: "performance/reports/bundle-stats.html",
-      template: "treemap",
-      sourcemap: true,
-    }) as PluginOption,
-    dts({
-      include: ["src"],
-      exclude: ["src/**/*.test.tsx", "src/**/*.test.ts"],
-      insertTypesEntry: true,
-      outDir: "dist/types",
-      tsconfigPath: "./tsconfig.json",
-      entryRoot: "src",
-    }),
-  ],
+import baseConfig from "./node_modules/@plyaz/devtools/configs/vite.config";
+
+baseConfig.plugins?.push(
```

Comment:
Maybe this would be good to have it across packages to check dependencies visualization